src/site/content/en/blog/aria-poison-or-antidote/index.md

```diff
@@ -0,0 +1,283 @@
+browser mode off. Screen readers are not generally necessary to exhibit a keyboard bug, and keyboard
+access is actually implemented with HTML, not ARIA. After all, ARIA doesn't affect basic stuff like
+the keyboard or mouse behavior, it only lies to the screen reader about what's in the web page,
+what's currently focused, etc. 
+
+Keyboard bugs are almost always a bug in the web content, specifically in their HTML and JavaScript,
+not in ARIA.
+
+### ARIA bugs… why are there so many?
+
+There are many, many places where authors can get ARIA wrong, and each will lead to either complete
+breakage or subtle differences. The subtle ones are probably worse, because the author won't catch
+most of them on their own.
+
+After all, unless the author is an experienced screen reader user, something is going to go wrong in
+the ARIA. In our menubar, the author could think the "option" role was to be used when "menuitem"
+was correct. They could forget to use `aria-expanded`, forget to set and clear
+`aria-activedescendant` at the right times, or forget to have a menubar containing the other menus.
+And what about menu item counts? Usually menu items are presented by screen readers with something
+like "item 3 of 5" so that the user knows where they are. This is generally counted automatically by
+the browser, but in some cases, and in some browser - screen reader combinations, the wrong numbers
+might be computed, and the author needs to override these numbers with `aria-posinset` and
+`aria-setsize`.
+
+And this is just menu bars. Think of how many kinds of widgets there are. Glance at the ARIA spec or
+authoring practices if you like. For each pattern, there are a dozen ways ARIA could be misused.
+ARIA relies on authors to know what they're doing. What could possibly go wrong, given that most
+authors are not screen reader users?
+
+In other words, it is 100% necessary for actual screen reader users to try ARIA widgets out before
+they're considered shippable. There's too much nuance. Ideally everything would be tried with
+several different browser - screen reader combinations, because of the numerous implementation
+quirks, in addition to a few incomplete implementations.
+
+### File bug on Chrome or on web content?
+
+For our team, which works on Chrome, we need to know when our representation of truth doesn't match
+what's provided by other browsers. When NVDA reads something different in the same web content in
+Firefox vs. Chrome, it's often our bug, since Firefox has an older, more seasoned accessibility
+mapping layer.
+
+If all the browser - screen reader combinations seem to agree, and the web content still behaves
+badly, we will need to file a bug on the web content itself, not on the browser.
+
+## Summary
+
+In summary, ARIA magic can be used to override anything and everything that the HTML says, or add to
+it. It can be used to do little fine changes to the accessibility presentation, or to create an
```

Comment:
```suggestion
In summary, ARIA magic can be used to override or add to anything and everything that the HTML says.
It can be used to do little fine changes to the accessibility presentation, or to create an
```